cpp/ps0002/ps0002_soln.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+
+int main(void)
+{
+  unsigned short residents = 50; // 0.5/0.75
+  unsigned short medicine = 20; // 0.75/0.75
+  unsigned short team = 50; // 0.75/0.75
+  float price = 10.5; // 0.75/0.75
+  char first_letter = 'M'; // 0.25/0.75
+  unsigned int china_populaltion = 120000000; // 0.5/0.75
+  long double division_7 = 0.142; // 0.5/0.75
+  float elec_invoice = 25000; // 0.5/0.75
+  unsigned int age_by_second = 6480000; // 0.5/0.75
+  // total 5/6.75
+
+  return 0;
+}
+// good work 👍 6/10
```

Comment:
- [ ] program is in the correct path
- [ ] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code